Jellyfin.Server.Implementations/Migrations/20201204223655_AddCustomDisplayPreferences.cs

```diff
@@ -0,0 +1,108 @@
+﻿#pragma warning disable CS1591
+// <auto-generated />
+using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Jellyfin.Server.Implementations.Migrations
+{
+    public partial class AddCustomDisplayPreferences : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropIndex(
+                name: "IX_DisplayPreferences_UserId_Client",
+                schema: "jellyfin",
+                table: "DisplayPreferences");
+
+            migrationBuilder.AlterColumn<int>(
```

Comment:
This is not compatible with SQLite?

Reply:
No idea, I just used the built in efcore migration

Reply:
Does it actually properly apply to a 10.6.0 DB and work as expected?

Reply:
EF Core 5 supports this for SQLite databases

Reply:
Yes, I copied my production 10.6 dbs and did all of the migrations